Makefile

```diff
@@ -84,5 +88,17 @@ njs-unit-test: ## Run unit tests for the njs httpmatches module
 		node:18 \
 		/bin/bash -c "npm install && npm test && npm run clean"
 
+.PHONY: generate-njs-yaml
+generate-njs-yaml: ## Generate the njs-modules ConfigMap
+	kubectl create configmap njs-modules --from-file=$(NJS_DIR)/httpmatches.js --dry-run=client --output=yaml > $(MANIFEST_DIR)/njs-modules.yaml
+
+.PHONY: fetch-crds-yaml
+fetch-crds-yaml: ## Fetch the Gateway API resources yaml and output it to the Helm chart crds folder
+	wget https://github.com/kubernetes-sigs/gateway-api/releases/download/v$(GW_API_VERSION)/standard-install.yaml -q -O $(CHART_DIR)/crds/gateway-crds.yaml
```

Comment:
shall we add any new tool required by a dev https://github.com/nginxinc/nginx-kubernetes-gateway/blob/ae2598fa6d2dce05ec40c39bbee76586724f8d51/docs/developer/quickstart.md#setup-your-development-environment ?

nit: curl is more adopted than wget :)